agenta-cli/agenta/sdk/agenta_init.py

```diff
@@ -95,15 +95,17 @@ def init(
 
 
 class Config:
-    def __init__(self, base_id: str, host: str, api_key: str = ""):
+    def __init__(self, base_id: str, host: str, api_key: Optional[str]):
```

Comment:
This function will result in an error if called with `base_id` and `host` only.

Reply:
No, it will not. The reason is that in the case of OSS, where the `api_key` is None, and that is what is passed to the `Config` object in the `AgentaSingleton` `init` method, we have a [condition](https://github.com/Agenta-AI/agenta/pull/1866/files#diff-b0f76a2e615c00fa9df3aa8266d52d1edd795fe45beb2fece84d35f831bb66cdR107) to check and use an empty string if the `api_key` resolves to None.

Reply:
ok, I was confused because of the Optional[str], usually optional means that there is a default value (that you can run the call without that paramter, which is not hte case here)